src/invidious/routes/misc.cr

```diff
@@ -40,7 +40,16 @@ module Invidious::Routes::Misc
 
   def self.cross_instance_redirect(env)
     referer = get_referer(env)
-    instance_url = fetch_random_instance
+
+    instance_list = Invidious::Jobs::InstanceListRefreshJob::INSTANCES["INSTANCES"]
```

Comment:
I guess this should be used to add some sites for reddit comments eventually as well https://github.com/iv-org/invidious/blob/9e8baa35397671aabfc77f6b912c9f1829be52b6/src/invidious/comments/reddit.cr#L8-L9

Reply:
Sounds good to me, this can be used for the eventual Lemmy source as well. 

Though does Reddit's search functionality even support filtering for that many sites at once?